examples/frame_prediction/js/entities/entities.js

```diff
@@ -0,0 +1,76 @@
+/************************************************************************************/
+/*                                                                                  */
+/*     Base entity that predicts its position based on the last update              */
+/*                                                                                  */
+/************************************************************************************/
+game.FramePredictionEntity = me.Entity.extend({
+   
+    init: function(x, y, settings) {
+        this._super(me.Entity, "init", [x, y , settings]);
+    },
+    
+    draw: function(renderer) {
+        // Predict the next position to draw at based on velocity
+        var updateInterval = (me.timer.getTime() - me.timer.lastUpdate) / ( (1000/me.sys.updatesPerSecond) );
```

Comment:
Right now the draw method is passed two arguments:

1. Renderer reference
1. Visible rectangle

I'm not sure how useful the rectangle is. For prediction purposes it would be ideal to just pass a delta time as the second argument, as done with the update method. @obiot @agmcleod Any opinions on this?